account_invoice_import_ubl/demo/demo_data.xml

```diff
@@ -1,82 +1,88 @@
-    </record>
-    <record id="nl_vat_purchase_21" model="account.tax">
-        <field name="name">Dutch VAT purchase 21.0%</field>
-        <field name="description">NL-VAT-buy-21.0</field>
-        <field name="type_tax_use">purchase</field>
-        <field name="amount">21</field>
-        <field name="amount_type">percent</field>
-        <field name="unece_type_id" ref="account_tax_unece.tax_type_vat" />
-        <field name="unece_categ_id" ref="account_tax_unece.tax_categ_s" />
-        <field name="account_id" ref="charge_account" />
-        <field name="refund_account_id" ref="charge_account" />
-    </record>
-    <record id="nl_vat_purchase_6" model="account.tax">
-        <field name="name">Dutch VAT purchase 6.0%</field>
-        <field name="description">NL-VAT-buy-6.0</field>
-        <field name="type_tax_use">purchase</field>
-        <field name="amount">6</field>
-        <field name="amount_type">percent</field>
-        <field name="unece_type_id" ref="account_tax_unece.tax_type_vat" />
-        <field name="unece_categ_id" ref="account_tax_unece.tax_categ_s" />
-        <field name="account_id" ref="charge_account" />
-        <field name="refund_account_id" ref="charge_account" />
-    </record>
-    <record id="exact_belgium" model="res.partner">
-        <field name="name">Exact Belgium II BVBA</field>
-        <field name="is_company" eval="True" />
-        <field name="supplier" eval="True" />
-        <field name="customer" eval="False" />
-        <field name="street">Koningin Astridlaan 166</field>
-        <field name="zip">1780</field>
-        <field name="city">Wemmel</field>
-        <field name="country_id" ref="base.be" />
-        <field name="vat">BE0505890632</field>
-    </record>
-    <record id="exact_belgium_import_config" model="account.invoice.import.config">
-        <field name="name">Exact Belgium</field>
-        <field name="partner_id" ref="exact_belgium" />
-        <field name="invoice_line_method">nline_no_product</field>
-        <field name="account_id" ref="charge_account" />
-    </record>
-    <record id="multi_tank" model="res.partner">
-        <field name="name">Multi Tank Card bv</field>
-        <field name="is_company" eval="True" />
-        <field name="supplier" eval="True" />
-        <field name="customer" eval="False" />
-        <field name="street">Postbus</field>
-        <field name="zip">2130 KB</field>
-        <field name="city">Hoofddorp</field>
-        <field name="country_id" ref="base.nl" />
-        <field name="vat">NL009276749B01</field>
-    </record>
-    <record id="multi_tank_import_config" model="account.invoice.import.config">
-        <field name="name">Multi Tank</field>
-        <field name="partner_id" ref="multi_tank" />
-        <field name="invoice_line_method">nline_no_product</field>
-        <field name="account_id" ref="charge_account" />
-    </record>
+
+<record id="charge_account" model="account.account">
```

Comment:
Can I ask you why the indentation changed? I'm surprised pre-commit allows this. Or did pre-commit do this itself?

Reply:
I see that the indentation was modified in the 13.0 port, and then restored in the 14.0 port. I don't mind having the indentation as it is now, but I think the refactoring should be squashed.